src/app/organization/holidays/view-holidays/view-holidays.component.html

```diff
@@ -0,0 +1,66 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="accent" *ngIf="holidayData.status.value !== 'Active'">
+    <fa-icon icon="lock-open"></fa-icon>&nbsp;&nbsp;
+    Activate
+  </button>
+  <button mat-raised-button color="primary" [routerLink]="['edit']">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+  <button mat-raised-button color="warn" (click)="deleteHoliday()">
+    <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+    Delete
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="header">
+          Name
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.name }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          From Date
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.fromDate | date }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          To Date
+        </div>
+
+        <div fxFlex="50%">
+          {{ holidayData.toDate | date }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          Repayments Scheduled To
+        </div>
+
+        <div fxFlex="50%" *ngIf="holidayData.repaymentsRescheduledTo !== undefined && holidayData.reschedulingType !== 1">
+          {{ holidayData.repaymentsRescheduledTo | date }}
+        </div>
+
+        <div fxFlex="50%" *ngIf="holidayData.repaymentsRescheduledTo === undefined && holidayData.reschedulingType === 1">
+          Next Repayment Date
```

Comment:
![Screenshot from 2020-05-22 18-51-45](https://user-images.githubusercontent.com/54709463/82671986-5b8bc680-9c5d-11ea-869c-2e9d9926d497.png)
Logic in community app is different why?

Reply:
I'll check and update the PR

Reply:
Done